app/app_client.go

```diff
@@ -1069,6 +1460,20 @@ func (c *AppClient) GetRobotMetadata(ctx context.Context, robotID string) (map[s
 }
 
 // UpdateRobotMetadata updates the user-defined metadata for a robot.
+//
+// UpdateRobotMetadata example:
+//
+//	err := cloud.UpdateRobotMetadata(
+//		context.Background(),
+//		"1ab2345c-a123-1ab2-1abc-1ab234567a12",
+//		map[string]interface{}{
+//			"key": "value",
+//		},
+//	)
+//
+// For more information, see the [UpdateRobotMetadata method docs].
+//
+// [UpdateRobotMetadata method docs]: https://docs.viam.com/dev/reference/apis/fleet/#updatemetadata
```

Comment:
should this be `https://docs.viam.com/dev/reference/apis/fleet/#updaterobotmetadata` instead?

Reply:
Good catch